cli/src/opts.rs

```diff
@@ -30,6 +30,10 @@ pub struct Opts {
     /// The path to a TOML file containing `local_server` configuration.
     #[arg(short = 'C', long = "config")]
     config_path: Option<PathBuf>,
+    /// Use Viceroy to run a module's _start function once, rather than in a
```

Comment:
Could you add something like `[EXPERIMENTAL]` to this docstring so it shows up in `viceroy --help` as such? That'll help hold us over until the followup PR.